source/overview/faq.rst

```diff
@@ -43,6 +43,7 @@ You can get involved and contribute to Mattermost in the following ways:
 - Find "Help Wanted" projects
 - Join Developer Discussion on a Mattermost Server for contributors
 - File Bugs
+- Quality Assurance and Testing
```

Comment:
The other items in the list start with a verb so maybe `Test new features` or something along those lines (?)